idaplugin/rematch/config.py

```diff
@@ -12,8 +12,12 @@ class Config(dict):
         "repository": "rematch",
```

Comment:
why are you creating a string here and not a dict ? you're just json'ifying everything later on and it would be easier to add elements if it were a dict.